apollo-router/src/configuration/expansion.rs

```diff
@@ -43,6 +45,16 @@ impl Expansion {
             .split(',')
             .map(|mode| mode.trim().to_string())
             .collect::<Vec<String>>();
+
+        let dev_mode_defaults = if std::env::var(APOLLO_ROUTER_DEV_ENV).ok().as_deref()
+            == Some("true")
+        {
+            tracing::info!("Running with *development* mode settings which facilitate development experience (e.g., introspection enabled)");
```

Comment:
should we link to a router docs "dev-mode" section?

Reply:
Good idea, but out of scope for this PR. We would need a stable URL set up.